src/index.js

```diff
@@ -1,16 +1,8 @@
-const express = require('express');
-const Server = require('./my-server');
-
-const app = express();
-
-app.use(express.static('static'));
+require('dotenv').config();
```

Comment:
Let's an empty line after this line and replace the `newline-after-var` ESlint rule with the [padding-line-between-statements](https://eslint.org/docs/rules/padding-line-between-statements) rule. I guess we can try to use `*` for both `prev` and `next` rule configuration options.